### DISCUSSION TITLE
The new legacy build question

### DISCUSSION THREAD

**Discussion by yarub123** [sereneblue/chameleon#554] [Q&A] [answered]
Is it JUST for those on older versions of FF? (older than v. 75) or should the newer versions of FF like 107.0 also update to the legacy one? I'm getting notified to update it. But checking the commits that have been made for the legacy build, they don't seem to be adding anything different besides Portuguese language for some people. 

Should newer versions just avoid the update? 

Thanks for the awesome work.

**Comment by sereneblue** [comment, marked as answer]
Some details about the legacy build can be found here: https://sereneblue.github.io/chameleon/wiki/legacy.

TLDR The only difference between the legacy build and the release version is that in the legacy build Chameleon assumes control of certain privacy settings (can't opt out). This version is really meant to be used by Android users still using Firefox v68.

**Reply by yarub123** [reply]
@sereneblue Ah okay thank you for clarifying. That makes much more sense. The original release notes I read on git didn't mention Android. But it is my fault, because there is a little button to click, "More info can be found here." All my dumbass had to do was click that. Thanks again.
